crates/c/src/lib.rs

```diff
@@ -2264,8 +2273,13 @@ impl Bindgen for FunctionBindgen<'_, '_> {
                         if !self.gen.in_import && self.gen.autodrop_enabled() {
                             // Here we've received a borrow of an imported resource, which is the
                             // kind we'll need to drop when the exported function is returning.
-                            let target = dealias(self.gen.resolve, *id);
-                            self.borrows.push((op.clone(), target));
+                            let ty = dealias(self.gen.resolve, *id);
+
+                            let name = self.locals.tmp("borrow");
+                            uwriteln!(self.borrow_decls, "int32_t {name} = 0;");
```

Comment:
Is there a better way to refer to the type of a handle here, or is using `int32_t` directly fine?

Reply:
Nah int32_t is fine, I don't think there's anything "more official" at this time